examples/simple/src/App.tsx

```diff
@@ -1,8 +1,47 @@
-import { Avatar, useAvatar } from 'alpha-ai-avatar-sdk-react';
+import {
+  Avatar,
+  TranscriberStatus,
+  useAvatar,
+} from 'alpha-ai-avatar-sdk-react';
 import { Button } from './Button';
+import { useEffect } from 'react';
 
 export function App() {
-  const { room, isConnected, connect, say, stop, switchAvatar } = useAvatar();
+  const {
+    room,
+    isConnected,
+    connect,
+    disconnect,
+    say,
+    stop,
+    messages,
+    switchAvatar,
+    enableMicrophone,
+    addEventListener,
+    removeEventListener,
+  } = useAvatar();
+
+  useEffect(() => {
+    const onTranscriberStatusChangeHandler = (status: TranscriberStatus) => {
+      console.log('Transcriber status changed:', status);
+    };
+
+    addEventListener(
+      'transcriberStatusChange',
+      onTranscriberStatusChangeHandler,
+    );
+
+    return () => {
+      removeEventListener(
+        'transcriberStatusChange',
+        onTranscriberStatusChangeHandler,
+      );
+    };
+  }, []);
```

Comment:
suggestion: instead of having event listener functions, the react way is to have a `transcriberStatus` state, and let the react components use `useEffect` to do something when the state changes if they need

Reply:
https://github.com/agencyenterprise/alpha-ai-avatar-sdk-react/pull/10/commits/931f36f53527b5792942f48d93a4e215a25203d9